src/components/KInputSwitch/KInputSwitch.vue

```diff
@@ -57,7 +57,7 @@ const props = defineProps({
   * Sets whether or not toggle is checked
   */
   modelValue: {
-    type: Boolean,
+    type: [Boolean, null],
```

Comment:
If we made this change, it would need to be `Null` not `null`

Applies to all instances

Reply:
It's `null` not `Null`

![Screenshot 2024-10-16 at 1 32 47 PM](https://github.com/user-attachments/assets/a6d84f9f-dd02-411f-8400-8030cf8beb0b)